vogue/server/endpoints/common_trends.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
+from flask import render_template, Blueprint, current_app, request
+
+from vogue.constants.constants import MONTHS, YEARS, THIS_YEAR
+from vogue.server.utils.metric_per_month import value_per_month
+from vogue import __version__
+
+app = current_app
+common_trends_blueprint = Blueprint('common_trends', __name__)
+
+
+@common_trends_blueprint.route('/common/turn_around_times/<year>')
+def turn_around_times(year):
+    y_vals = [
+        'received_to_delivered', 'received_to_prepped', 'prepped_to_sequenced',
+        'sequenced_to_delivered'
+    ]
+    results_grouped_by_prio = {}
+    results_grouped_by_cat = {}
+    for y_val in y_vals:
+        results_grouped_by_prio[y_val] = value_per_month(
+            app.adapter, year, y_val, "priority")
```

Comment:
Suggest to name the arguments to avoid bugs sneaking in like
```suggestion
        results_grouped_by_prio[y_val] = value_per_month(
            adapter=app.adapter, year=year,y_axis= y_val, x_axis="priority")
```
(I don't know the actual parameter names)